core/src/main/scala/akka/kafka/internal/ConsumerStage.scala

```diff
@@ -141,12 +156,13 @@ private[kafka] object ConsumerStage {
 
   final case class CommittableOffsetImpl(override val partitionOffset: ConsumerMessage.PartitionOffset)(val committer: Committer)
       extends CommittableOffset {
-    override def commitScaladsl(): Future[Done] = committer.commit(partitionOffset)
+    override def commitScaladsl(): Future[Done] = committer.commit(Seq(partitionOffset))
     override def commitJavadsl(): CompletionStage[Done] = commitScaladsl().toJava
   }
 
   trait Committer {
-    def commit(offset: PartitionOffset): Future[Done]
+    // Commit all offsets (of different topics) belonging to the same stage
+    def commit(offsets: Seq[PartitionOffset]): Future[Done]
```

Comment:
`Seq` can be mutable, e.g. `Array`, therefore we explicitly use the `immutable` one by:
```scala
import scala.collection.immutable

def commit(offsets: immutable.Seq[PartitionOffset]): Future[Done]
```